1.segment_images/scripts/0.segment_nuclei_organoids.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This notebook focuses on trying to find a way to segment cells within organoids properly.
+# The end goals is to segment cell and extract morphology features from cellprofiler.
+# These masks must be imported into cellprofiler to extract features.
```

Comment:
Recommend including more info on the 2.5D methodology, it is the most important part here. You might also consider moving it out into a README instead. Here is my suggestion:

```
For segmentation, we have found 3D too computationally expensive and yields less than optimal results. 
We have developed a 2.5D segmentation methodology in combination with Cellpose.
For nuclei, we take a sliding window of 3 z-slices, where we perform max-projection on these slides and perform segmentation on these slices.
We then decouple the max-projection to collect a segmentation mask per z-slice.
We then output the masks across all z-slices per image set as a stacked-TIFF file.
```